sample-prebid-config.yaml

```diff
@@ -0,0 +1,23 @@
+adapters:
```

Comment:
What about moving these two sample files to `sample` directory or alike?

Reply:
The idea was to make it simpler for newcomer for quick start, so it is better these files be in root folder.